knowledge-base/chart-localized-labels.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Localize numeric labels in the Chart
+description: How to localize numeric labels in the Chart
+type: how-to
+page_title: Localize numeric labels in the Chart
+slug: chart-kb-localized-numeric-labels
+position:
+tags:
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Chart for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+I am having a Chart and I would like my labels to be localization aware.
+
+
+## Solution
+
+There are two different approaches:
+
+* [Using the Format Parameter](#using-the-format-parameter)
+* [Using the Template Parameter](#using-the-template-parameter)
+
+### Using the Format Parameter
+
+If you require your label to be a number only, you can use the [`Format`]({%slug components/chart/label-template-format%}#format-strings) parameter of the `<ChartSeriesLabels>` tag. It takes a [standard numeric format strings](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings) and is localization aware.
```

Comment:
Test if this works only for numbers. I suspect something like `Format="lorem ipsum {0:C}"` will work too, so maybe we can reword to something like "if you need to use only the Value of the series item, and a single line of text, you can use the format"

Reply:
It indeed does, so I have done the wording as suggested and also reword a little the entry paragraph for the solution that uses the Template.